LibTrixi.jl/src/api_jl.jl

```diff
@@ -106,3 +106,27 @@ function trixi_load_cell_averages_jl(data, simstate)
 
     return nothing
 end
+
+function trixi_version_major_jl()
+    return VersionNumber(_version_libtrixi[]).major
+end
+
+function trixi_version_minor_jl()
+    return VersionNumber(_version_libtrixi[]).minor
+end
+
+function trixi_version_patch_jl()
+    return VersionNumber(_version_libtrixi[]).patch
+end
```

Comment:
If we decide to keep those functions (see my other comment below), they should return `Int`s and not `UInt32` imho:
```suggestion
function trixi_version_major_jl()
    return Int(VersionNumber(_version_libtrixi[]).major)
end

function trixi_version_minor_jl()
    return Int(VersionNumber(_version_libtrixi[]).minor)
end

function trixi_version_patch_jl()
    return Int(VersionNumber(_version_libtrixi[]).patch)
end
```